### DISCUSSION TITLE
Welcome to nextjs-corporate-starter Discussions!

### DISCUSSION THREAD

**Discussion by PaulBratslavsky** [strapi/nextjs-corporate-starter#34] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Maxch3306** [comment]
Hello may I know how to use i18n in this package
I saw the i18n is built but I add the lang in Strapi but it doesn't affect the frontend

sorry I should correct my question: the global data/blog pages is not affected by i18n
